modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -19,52 +19,51 @@ A new message flow is created. +
 
 . Expand the *Source at <host-name>* section.
 . In *Endpoint:Source at <host-name>*, click *Select*.
-. To use an existing receiving endpoint: 
+. To use an existing receiving endpoint:
 .. Select the endpoint from the list and click *Select* in the lower right side of the page. +
-The endpoint is added to the message flow, and you see a message that tells you the endpoint is missing an identifier. 
+The endpoint is added to the message flow, and a message appears indicating that the endpoint is missing an identifier.
 .. Click *Add* to add a unique identifier for the endpoint.
 . To create a new receiving endpoint:
-.. Click *New* on the right of the search box. 
-.. From the *Owner* drop-down, select the endpoint's owner. 
+.. Click *New* on the right of the search box.
+.. From the *Owner* drop-down, select the endpoint's owner.
 .. From the *Protocol* drop-down, select the protocol and configure it according to the protocol you select:
 * xref:endpoint-https-send.adoc[*HTTP(S)*]
 * xref:endpoint-sftp-send.adoc[*SFTP*]
 . Click *Save*. +
-The endpoint is added to the message flow. 
+The endpoint is added to the message flow.
 
 == Configure the Source Message Type
 
 . Expand the *Source message type* section, and click *Select*.
-. To use an existing message type:
-.. Select the message type from the list and click *Select* in the lower right side of the page. +
-The message type is added to the message flow. 
-. To create a new message type: 
+. To use an existing message type, select the message type from the list and click *Select* in the lower right side of the page. +
```

Comment:
Here is yet a third treatment of these two options:
-"alternatively" plain text in one step
-substeps for each option
-separate steps for each option
Please pick one treatment and standardize to it throughout.
Consider text between title and Step 1 that prepares audience for either creating a new or editing an existing, throughout.